### PR TITLE
docs(docs-site): featuring subquery in docs

### PR DESCRIPTION
**Pull request by sq-ext**

### CODE REVIEW

File: packages/docs-site/src/content/docs/guides/indexers/overview.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Overview
+description: An overview of indexers
+---
+
+Indexers play a crucial role in organising and optimising data retrieval across various systems. They serve as navigational guides, enhancing the efficiency of data access by creating structured indexes. Within the domain of databases and information management, indexers improve query performance by providing a roadmap to swiftly locate specific blockchain data entries.
+
+In the context of blockchain and decentralised applications, indexers take on an even more pivotal role. They facilitate streamlined access to on-chain data, encompassing transaction histories, smart contract states, and event logs. In the dynamic and decentralised blockchain ecosystem, indexers ensure real-time data updates and seamless functionality for an array of applications and platforms.
```
Comment (swarna1101) [inline packages/docs-site/src/content/docs/guides/indexers/overview.md:8]: Specify the importance of real-time data and efficiency in the context of Taiko's architecture.

File: packages/docs-site/src/content/docs/guides/indexers/subquery.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: SubQuery
+description: Indexing Taiko with SubQuery
+---
+
+SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports many ecosystems including Taiko, Ethereum, Cosmos, Near, Polygon, Polkadot, Algorand, and more.
```
Comment (swarna1101) [inline packages/docs-site/src/content/docs/guides/indexers/subquery.md:6]: Highlight specific features of SubQuery that are particularly beneficial for Taiko’s ZK-Rollup architecture, such as handling high throughput and low latency.

File: packages/docs-site/src/content/docs/guides/indexers/subquery.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- [Taiko Holesky Starter](https://github.com/subquery/ethereum-subql-starter/tree/main/Taiko/taiko-holesky-starter)
+- [Taiko Katla Starter](https://github.com/subquery/ethereum-subql-starter/tree/main/Taiko/taiko-katla-starter)
+- [Quick Start Guide](https://academy.subquery.network/indexer/quickstart/quickstart.html)
+
+For technical questions and support reach out to us `start@subquery.network`
+
+## Running and Hosting Taiko SubQuery APIs
+
+SubQuery is open-source, meaning the developers have the freedom to run it in the following three ways:
+
```
Comment (swarna1101) [inline packages/docs-site/src/content/docs/guides/indexers/subquery.md:22]: Emphasize the flexibility and control developers have when choosing how to run SubQuery

File: packages/docs-site/src/content/docs/guides/indexers/subquery.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- Locally (or a cloud provider), [view the instructions on how to run SubQuery Locally](https://academy.subquery.network/run_publish/run.html).
+- SubQuery's enterprise-level [Managed Service](https://managedservice.subquery.network/), where the SubQuery project will be hosted in production ready services for mission critical data with zero-downtime blue/green deployments. There even is a generous free tier. [Find out how](https://academy.subquery.network/run_publish/publish.html).
+- Decentralised [SubQuery Network](https://subquery.network/network), the most open, performant, reliable, and scalable data service for dApp developers. The SubQuery Network indexes and services data to the global community in an incentivised and verifiable way and supports Taiko from launch.
```
Comment (swarna1101) [inline packages/docs-site/src/content/docs/guides/indexers/subquery.md:25]: Provide specific examples of how SubQuery’s decentralized network benefits Taiko, such as improved data availability and robustness.

**Review by swarna1101 — CHANGES_REQUESTED** [review]

**Review by dantaik — APPROVED** [review]

**Comment by d1onys1us** [comment]
after discussing with @RogerLamTd this feels a bit off for the core docs. i would recommend adding to the ecosystem page.

**Pull request closed without merging**